Cargo.toml

```diff
@@ -15,10 +15,10 @@ keywords = ["metrics", "cloudwatch", "aws"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-chrono = "0.4.9"
-futures = "0.1.29"
+chrono = "0.4.10"
+futures = { version = "0.3.4", features = ["compat"] }
```

Comment:
Is compat needed if you are using rusutos beta?

Reply:
Good catch!